python/runtime/step/tensorflow/evaluate.py

```diff
@@ -45,6 +45,7 @@ def evaluate_step(datasource,
                   model,
                   label_name,
                   model_params,
+                  result_column_names=[],
```

Comment:
Need to remove this line: 

https://github.com/sql-machine-learning/sqlflow/blob/5e7f937bcf872a1bd7fe6e1ef2568230df1403c9/python/runtime/step/tensorflow/evaluate.py#L65